5_People/benefits.md

```diff
@@ -96,8 +96,7 @@ To book vacation time off, add an entry to the [Planned Absences](https://github
 
 ### Unused Vacation
 
-Unused vacation hours in a calendar year can be carried over to the next year but must be used by the 1st of July. If not, we are going to have a serious conversation because everyone needs a break.
-
+Unused vacation hours in a calendar year can be carried over to the next year but must be used by the 1st of July otherwise you will forfeit them.
```

Comment:
I much preferred the old narrative. The new one tells a story that your hours will be "stolen" instead of the one where *everyone* needs a break and you should be responsible to take breaks.

Reply:
@zupo The old one "encourages" Niteans without actual consequences, while this one really pushes Niteans spend their vacation hours and take breaks (which is what we really want).

Reply:
@zupo Maybe 👇 where we keep the old narrative but with an addendum

> Unused vacation hours in a calendar year can be carried over to the next year but must be used by the 1st of July. If not, we are going to have a serious conversation because everyone needs a break. 
> 
> Scrooge does a cleanup up all remaining hours from the previous year every July.

Reply:
Unused vacation hours in a calendar year can be carried over to the next year but must be used by the 1st of July. Please make sure to take enough time off to spend your vacation hours before this deadline, we hate taking them away from you!

Reply:
?